eq_translations/validate_translation.py

```diff
@@ -0,0 +1,103 @@
+from jsonpointer import resolve_pointer
+from termcolor import colored
+
+from eq_translations.utils import (
+    list_pointers,
+    find_pointers_to,
+    get_plural_forms_for_language,
+)
+
+
+def compare_schemas(source_schema, target_schema):
+    """
+    Compare the pointers in two json structures and return differences
+    :param source_schema: Structure to identify differences against
+    :param target_schema: Target structure to compare against
+    :return:
+    """
+    non_plural_source_survey_pointers = {
+        p for p in list_pointers(source_schema) if "text_plural" not in p
+    }
+    non_plural_target_survey_pointers = {
+        p for p in list_pointers(target_schema) if "text_plural" not in p
+    }
+
+    missing_target_pointers = non_plural_source_survey_pointers.difference(
+        non_plural_target_survey_pointers
+    )
+    missing_source_pointers = non_plural_target_survey_pointers.difference(
+        non_plural_source_survey_pointers
+    )
+
+    missing_pointers = missing_target_pointers | missing_source_pointers
+
+    if missing_pointers:
+        print(
+            colored(
+                f"\nDifferences between source/target schema strings (excluding text_plural): {len(missing_pointers)}",
+                "red",
+            )
+        )
+        print(
+            f"Total non-plural strings in source schema: {len(non_plural_source_survey_pointers)}"
+        )
+        print(
+            f"Total non-plural strings in target schema: {len(non_plural_target_survey_pointers)}\n"
+        )
+
+        if missing_target_pointers:
+            print(
+                colored(
+                    "Pointers in the source schema that are missing from the target schema:",
+                    "red",
```

Comment:
- It would be more helpful if this wasn't in red, as it is part of the "Differences" output. It could just be added on to the lines above, without a new line. 
- Using some list type formatting with `-` and spaces helps

Something like:

```
Differences between source/target schema strings (excluding text_plural): 1
Total non-plural strings in source schema: 130
Total non-plural strings in target schema: 129
Missing from the target schema:
  - /sections/0/groups/0/blocks/0/question/title

Total plural forms missing in translated schema: 1
Missing plural forms at:
 - /sections/0/groups/0/blocks/3/question/title/text_plural/forms/: 'few'
```